common/OpTestSystem.py

```diff
@@ -1002,6 +1002,8 @@ def sys_ipmi_boot_system_to_petitboot(self, i_console):
         else:
             l_msg = "System failed to reach standby/Soft-off state"
             raise OpTestError(l_msg)
+
+        self.cv_IPMI.ipmi_set_boot_to_petitboot()
```

Comment:
I also faced same issue when executing the pci scan test. system booted with existing os before it exit to the petitboot shell. this solution will work good, but earlier i raised some bugs on this, once we set a ipmi boot device it is persistent across boots. Then by this way we can catch that bug too